examples/ContainedTable.example.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  createStyles,
+  PropsFromStyles,
+  TableContainer,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableHeaderCell,
+  TableBodyCell,
+} from 'hacker-ui';
+
+const useStyles = createStyles(({ css, theme }) => ({
+  root: css`
+    margin-top: ${theme.space(1)};
+  `,
+  numberCell: css`
+    text-align: right;
+  `,
+}));
+
+interface Props extends PropsFromStyles<typeof useStyles> {}
+
+function createData(
+  name: string,
+  numberPerOnce: number,
+  calories: number,
+  protein: number,
+  totalFat: number,
+) {
+  return {
+    name,
+    numberPerOnce,
+    calories,
+    protein,
+    totalFat,
+  };
+}
+
+const rows = [
+  createData('Almonds', 23, 160, 6, 14),
```

Comment:
small thing: I think it would be simpler to have arrays of objects instead of the `createData` function

Reply:
I think that the createData function is pretty simple and it makes it very clear the content & type of the data.